lib/transmuxer/version.rb

```diff
@@ -1,3 +1,3 @@
 module Transmuxer
-  VERSION = "1.0.1"
+  VERSION = "1.0.3"
```

Comment:
Why not 1MILLION muahaha. I suppose it's arbitrary. It didn't seem like a major overhaul to me. No preference though personally.